ui/page/components/components.go

```diff
@@ -1,4 +1,4 @@
-// components contain layout code that are shared by multiple pages but aren't widely used enough to be defined as
+// components.go contain layout code that are shared by multiple pages but aren't widely used enough to be defined as
```

Comment:
Kindly revert this... the comments is talking about the component package not the file.